integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -5337,27 +5372,29 @@ msgstr ""
 "Nicht (⚠️) veröffentlichte und veröffentlichte Seiten für die "
 "%(language_translated_name)s-Übersetzung exportieren"
 
-#: cms/templates/pages/page_tree.html:186
+#: cms/templates/pages/page_tree.html:191
 msgid "Export selected pages for multilingual translation"
-msgstr "Ausgewählte Seiten für die mehrsprachige Übersetzung exportieren"
+msgstr ""
+"Ausgewählte Seiten für die %(language_translated_name)s-Übersetzung "
```

Comment:
In this part of the tree template we don't have access to the `language_translated_name` variable.

Reply:
Thanks! Not exactly sure how this translation got changed...